api/hrlh.rb

```diff
@@ -17,7 +18,17 @@ def interview_db
       end
 
       def to_interviewer(result)
-        result.value.merge({"key"=>result.key})
+        interviewer = result.value.merge({"key"=>result.key})
```

Comment:
Redundant curly braces around a hash parameter.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.